mne/viz/utils.py

```diff
@@ -2516,6 +2516,8 @@ def _set_title_multiple_electrodes(title, combine, ch_names, max_chans=6,
         title = ", ".join(ch_names[:max_chans])
         if ch_type is not None:
             ch_type = " " + ch_type[0].upper() + ch_type[1:]
```

Comment:
What the hell did I do here, and why? Why would I want to write "Eeg"?

Reply:
No idea why it is like this

Reply:
This seems fine. Are you thinking of `capitalize`?
```
>>> 'foo'.upper()
'FOO'
>>> 'foo'.capitalize()
'Foo'
```

Reply:
Yes, but why would we want to read Meg or Eeg?..

Reply:
Ok, out of these:
`['eeg', 'grad', 'mag', 'seeg', 'eog', 'ecg', 'emg',
 'dipole', 'gof', 'bio', 'ecog', 'hbo', 'hbr',
 'misc']`
we probably want EEG and EMG capitalised, misc should be misc., mag should be Magnetometers .... We don't have a dict mapping between these two, don't we?